src/__test__/initial.spec.ts

```diff
@@ -70,8 +65,19 @@ describe("test a user signup endpoint", () => {
         expect(res).to.have.status(400);
       });
   });
-});
 
+  // it("user fail to verify user with invalid token", () => {
```

Comment:
Why did you comment out this test case?

Reply:
I brought it back, and now it is working well. i was checking why it was not working on circle ci but now i have fixed it.